regression/regression_param.sh

```diff
@@ -117,17 +117,17 @@ case $regtest in
     hafs_3denvar_hybens)
 
         if [[ "$machine" = "Hera" ]]; then
-           topts[1]="0:15:00" ; popts[1]="20/1/"  ; ropts[1]="/1"
-           topts[2]="0:15:00" ; popts[2]="20/2/"  ; ropts[2]="/1"
+           topts[1]="0:15:00" ; popts[1]="5/4/"  ; ropts[1]="/1"
+           topts[2]="0:15:00" ; popts[2]="10/4/"  ; ropts[2]="/1"
         elif [[ "$machine" = "Orion" ]]; then
-           topts[1]="0:15:00" ; popts[1]="20/1/" ; ropts[1]="/1"
-           topts[2]="0:15:00" ; popts[2]="20/2/" ; ropts[2]="/2"
+           topts[1]="0:15:00" ; popts[1]="5/4/" ; ropts[1]="/1"
+           topts[2]="0:15:00" ; popts[2]="10/4/" ; ropts[2]="/2"
         elif [[ "$machine" = "Hercules" ]]; then
            topts[1]="0:15:00" ; popts[1]="20/1/" ; ropts[1]="/1"
-           topts[2]="0:15:00" ; popts[2]="20/2/" ; ropts[2]="/2"
+           topts[2]="0:15:00" ; popts[2]="5/8/" ; ropts[2]="/1"
```

Comment:
Should the number of threads be 1 for all `hafs_3denvar_hybens` tests?  If so, then should Orion have its `hiproc` thread count reduced to match Hercules?

Reply:
@DavidHuber-NOAA , now, I changed those threads setup in this PR to the current head of EMC GSI has because I don't have any reasons to change them.    Since there are changes related to hercules too, I will re-run regression tests on hercules.

Reply:
BTW:  the regression tests with the current popts setup for hera have all passed on hera (including that hafs_4densvar_glbens 's failing for "exceeded maximum allowable threshhold time.." was ignored)